pkgs/development/python-modules/parsley/default.nix

```
@@ -0,0 +1,20 @@
+{ buildPythonPackage
+, fetchPypi
+, stdenv
```

personal preference to avoid stdenv when possible.
```suggestion
, llib
```

---

fixed